wikimedia/wikipedia-ios
# Update names of Swift Packages

**Phabricator:** https://phabricator.wikimedia.org/T371736

### Notes
General naming updates for our Swift Packages. Note I turned off the old package unit tests for now. I think the new names will need to be merged into `main` before I can bring them back.

### Test Steps
1. Confirm main Wikipedia scheme unit tests pass in CI, and do light regression testing. Locally you can confirm the data and components schemes pass as well.
